api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -134,6 +136,10 @@ message BufferSettings {
   // The authorization request will be dispatched and no 413 HTTP error will be returned by the
   // filter.
   bool allow_partial_message = 2;
+
+  // If true, the body sent to the external authorization service is set with raw bytes, instead of
+  // UTF-8 string.
```

Comment:
In the docs can you clarify which field gets filled in the request depending on how this is is set? I'm not sure we generate RST for the request so you might not be able to ref link (please do if possible) but otherwise you can just describe it manually. Same for the other location.